commit 562f18cdddc847e84ce0f73da535b996f8f0fd45
Author: Tobias Doerffel <tobias.doerffel@gmail.com>
Date:   Wed Dec 11 23:28:37 2013 +0100

    ItalcVncServer: allocate rfbProtocolExtension statically
    
    Otherwise "pe" is only valid within the if block and thus after calling
    rfbRegisterProtocolExtension the passed pointer is not valid anymore and
    causes a crash on recent GCCs.

diff --git italc-2.0.0.orig/ica/src/ItalcVncServer.cpp italc-2.0.0/ica/src/ItalcVncServer.cpp
index 65b4d8e..73e0ee7 100644
--- italc-2.0.0.orig/ica/src/ItalcVncServer.cpp
+++ italc-2.0.0/ica/src/ItalcVncServer.cpp
@@ -2,7 +2,7 @@
  * ItalcVncServer.cpp - implementation of ItalcVncServer, a VNC-server-
  *                      abstraction for platform independent VNC-server-usage
  *
- * Copyright (c) 2006-2010 Tobias Doerffel <tobydox/at/users/dot/sf/dot/net>
+ * Copyright (c) 2006-2013 Tobias Doerffel <tobydox/at/users/dot/sf/dot/net>
  *
  * This file is part of iTALC - http://italc.sourceforge.net
  *
@@ -273,7 +273,7 @@ static void runX11vnc( QStringList cmdline, int port, bool plainVnc )
 	if( plainVnc == false )
 	{
 		// register iTALC protocol extension
-		rfbProtocolExtension pe;
+		static rfbProtocolExtension pe;
 		pe.newClient = italcCoreNewClient;
 		pe.init = NULL;
 		pe.enablePseudoEncoding = NULL;
